message-ranges.txt

```diff
@@ -0,0 +1,107 @@
+# This document is intended to establish the ranges for message IDs in the various sub-projects.
+
+# Range     = owner
+
+# Provider (smallrye-reactive-messaging-provider)
+00000-00099 = exceptions
```

Comment:
It's odd to me that exceptions and messaging are separated, as they're the same kind of thing

Reply:
In this case, yes, because it's mostly use with `Objects.require...` that end up in an exception.
However one could translate something that is not in the log or an exception. However, if you want me to change it, I'll happily do so :) Just let me know.